packages/react-components/src/components/table-output-component.tsx

```diff
@@ -88,20 +88,22 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                 params.successCallback(rowsThisPage, this.props.nbEvents);
             }
         };
-
         this.onEventClick = this.onEventClick.bind(this);
         this.onModelUpdated = this.onModelUpdated.bind(this);
         this.onKeyDown = this.onKeyDown.bind(this);
         this.searchEvents = this.searchEvents.bind(this);
         this.findMatchedEvent = this.findMatchedEvent.bind(this);
+        this.checkFocus = this.checkFocus.bind(this);
     }
 
     renderMainArea(): React.ReactNode {
         if (this.state.outputStatus === ResponseStatus.FAILED) {
             return this.analysisFailedMessage();
         }
 
-        return <div id='events-table'
+        return <div id={this.props.outputDescriptor.id+'chartContainer'}
```

Comment:
Not a big deal, but chartContainer isn't appropriate for this component that only has a table.